corgidrp/l2b_to_l3.py

```diff
@@ -30,6 +32,44 @@ def divide_by_exptime(input_dataset):
 
     return input_dataset.copy()
 
+def determine_color_cor(input_dataset, calspec_filepath, source_filepath):
+    """
+    determine the color correction factor of the observed source
+    at the reference wavelength of the used filter band and put it into the header.
+    We assume that each frame in the dataset was observed with the same color filter.
+    
+    Args:
+        input_dataset (corgidrp.data.Dataset): a dataset of Images (L2b-level)
+        calspec_filepath (str): file name of the known reference flux (usually CALSPEC)
```

Comment:
Well, my imagination was, that we need to observe at least 2 stars (I guess these will be mostly cal standards with available calspec files, or at least known stellar models in calspec style format) one with and one without the ND filter in different color filter bands. I think there might well be different reference stars needed for the different bands, visibility times etc, so this should be flexible. I think the star observed without ND filter is the reference, and the observation of a star with ND filter will be calibrated on that, but it could of course also be vice versa.